From b12f6142f356a72af64665e753c5a4e61b2a6ec1 Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Thu, 8 Apr 2010 13:47:26 +1000
Subject: [PATCH 11/25] xfree86: remove dead input drivers from xorg.conf man page.

These drivers have been deactivated for over a year now, let's not refer
potential users to them.

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
(cherry picked from commit 03ccbd2579948d11e5f16f88cdf68a55f57c9d26)

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
---
 hw/xfree86/doc/man/xorg.conf.man.pre |    9 ---------
 1 files changed, 0 insertions(+), 9 deletions(-)

diff --git xorg-server-1.8.0/hw/xfree86/doc/man/xorg.conf.man.pre xorg-server-1.8.0/hw/xfree86/doc/man/xorg.conf.man.pre
index 4782d61..f7ff6f6 100644
--- xorg-server-1.8.0/hw/xfree86/doc/man/xorg.conf.man.pre
+++ xorg-server-1.8.0/hw/xfree86/doc/man/xorg.conf.man.pre
@@ -2405,25 +2405,16 @@ Display drivers:
 .PP
 Input drivers:
 .BR acecad (__drivermansuffix__),
-.BR calcomp (__drivermansuffix__),
 .BR citron (__drivermansuffix__),
-.BR dmc (__drivermansuffix__),
-.BR dynapro (__drivermansuffix__),
 .BR elographics (__drivermansuffix__),
 .BR evdev (__drivermansuffix__),
 .BR fpit (__drivermansuffix__),
-.BR js_x (__drivermansuffix__),
 .BR joystick (__drivermansuffix__),
 .BR kbd (__drivermansuffix__),
-.BR magictouch (__drivermansuffix__),
-.BR microtouch (__drivermansuffix__),
 .BR mousedrv (__drivermansuffix__),
 .BR mutouch (__drivermansuffix__),
-.BR palmax (__drivermansuffix__),
 .BR penmount (__drivermansuffix__),
 .BR synaptics (__drivermansuffix__),
-.BR tek4957 (__drivermansuffix__),
-.BR ur98 (__drivermansuffix__),
 .BR vmmouse (__drivermansuffix__),
 .BR void (__drivermansuffix__),
 .BR wacom (__drivermansuffix__).
-- 
1.7.0.4

